src/api/control.ts

```diff
@@ -30,6 +30,7 @@ keysMap.set("Shift+Escape", "home");
 keysMap.set("Control+Escape", "home");
 keysMap.set("Backspace", "instantreplay");
 keysMap.set("End", "play");
+keysMap.set("Digit8", "info");
```

Comment:
This change should be done in the `app/index.js` as a custom key map for the browser, there are already examples there.